### DISCUSSION TITLE
Missing entities using Kiosk

### DISCUSSION THREAD

**Discussion by supm3n** [tijsverkoyen/HomeAssistant-FusionSolar#182] [General]
I enabled Kiosk on the web portal, but home assistant is only getting the following entities:
x_realtime_power
x_total_current_day_energy
x_total_current_month_energy
x_total_current_year_energy
x_total_lifetime_energy

I am completely missing grid and battery entities. 
Can someone help?

**Comment by tijsverkoyen** [comment]
Kiosk is limited in the data that is exposed. The entities you have listed are the once that are exposed.
If you want more data I advice to use the Northbound / OpenAPI implementation.
